summarization/stamp_page_picking/max_cover_preprocessor.py

```diff
@@ -60,7 +60,7 @@ def _get_cost_for_stamp_pages(self):
             # block will be amended to add supporting logic
             # for deciding costs based on stamp page
             # type and content present
-            costs.append(self.stamp_costs_dict[stamp_page.stamp_type.name])
+            costs.append(1/self.stamp_costs_dict[stamp_page.stamp_type.name])
```

Comment:
cost (penalty) is inversely proportional to score for the stamp page.

Reply:
But we are not returning scores with `self.stamp_costs_dict[stamp_page.stamp_type.name]`, right?

Reply:
Ah yes, maybe we can change the naming to reflect that!